### PR TITLE
Update "Start a new React project" for stable Next.js App Router

### PR DESCRIPTION
**Pull request by leerob**
Should we merge the Next.js sections? Or keep as is? Open to either.

### CODE REVIEW

**Comment by gayashan4lk** [comment]
Duplicate pull request #6005

**Comment by lunaleaps** [comment]
We are revisiting this page. Perhaps we won't break it out as "bleeding-edge". https://github.com/reactjs/react.dev/pull/6318 already removes the outdated pitfall -- do you want to rebase and we can land your other updates?

File: src/content/learn/start-a-new-react-project.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ If you want to build a new app or a new website fully with React, we recommend p
...
 
 <TerminalBlock>
-npx create-next-app
+npx create-next-app@latest
 </TerminalBlock>
 
 If you're new to Next.js, check out the [Next.js tutorial.](https://nextjs.org/learn/foundations/about-nextjs)
 
-Next.js is maintained by [Vercel](https://vercel.com/). You can [deploy a Next.js app](https://nextjs.org/docs/deployment) to any Node.js or serverless hosting, or to your own server. [Fully static Next.js apps](https://nextjs.org/docs/advanced-features/static-html-export) can be deployed to any static hosting.
+Next.js is maintained by [Vercel](https://vercel.com/). You can [deploy a Next.js app](https://nextjs.org/docs/pages/building-your-application/deploying) to any Node.js or serverless hosting, or to your own server. Next.js also supports [static export](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports) which doesn't require a server.
```
Comment (leerob) [inline src/content/learn/start-a-new-react-project.md:29]:
```suggestion
Next.js is maintained by [Vercel](https://vercel.com/). You can [deploy a Next.js app](https://nextjs.org/docs/app/building-your-application/deploying) to any Node.js or serverless hosting, or to your own server. Next.js also supports [static export](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports) which doesn't require a server.
```

**Comment by leerob** [comment]
Updated!

**Pull request merged**